src/Jellyfin.Drawing.Skia/StripCollageBuilder.cs

```diff
@@ -22,6 +23,9 @@ public StripCollageBuilder(SkiaEncoder skiaEncoder)
         _skiaEncoder = skiaEncoder;
     }
 
+    [GeneratedRegex(@"\s{IsArabic}|\s{IsArmenian}|\s{IsHebrew}|\s{IsSyriac}|\s{IsThaana}", RegexOptions.IgnoreCase)]
```

Comment:
```suggestion
    [GeneratedRegex(@"\p{IsArabic}|\p{IsArmenian}|\p{IsHebrew}|\p{IsSyriac}|\p{IsThaana}")]
```

Change `\s` to `\p`
Remove `RegexOptions.IgnoreCase` as it has no effect.

Reply:
And remove the hard-coded width of 250